server/webapp/WEB-INF/vm/admin/server_version.txt.vm

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You probably want to populate a default file with actual keys and values set to `unknown` or something similar.